src/arrayMethodSort.js

```diff
@@ -4,9 +4,40 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultSortedAsStrings) {
+    let count;
+
+    do {
+      count = 0;
+
+      for (let i = 1; i < this.length; i++) {
+        const prev = this[i - 1];
+        const current = this[i];
+
+        if (compareFunction(prev, current) > 0) {
+          count++;
+
+          this[i - 1] = current;
+          this[i] = prev;
+        }
+      }
+    } while (count > 0);
+
+    return this;
   };
+
+  function defaultSortedAsStrings(element1, element2) {
+    const string1 = element1.toString();
```

Comment:
On my opinion, use structure's name in variable's name is bad practice. Once time, Vova reject my task, because i was used a structure's name in my variable's name)

Reply:
Agree with Igor, let's change change names for proper one's